docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md

```diff
@@ -0,0 +1,88 @@
+- [Notary v2](https://github.com/notaryproject/notation)
+- [KubeApps](https://github.com/vmware-tanzu/kubeapps) by VMware Tanzu
+- [VMware Application Catalog](https://tanzu.vmware.com/application-catalog)
+- [UOR Framework](https://universalreference.io/) by Red Hat
+- [soci-snapshotter](https://github.com/awslabs/soci-snapshotter) by AWS
+- [Zot](https://github.com/project-zot/zot)
+
+## Contributions to upstream OCI
+
+Just a few years ago, there were no standards nor tooling for registries to natively store, discover, and pull a graph of OCI artifacts. To extend the registry’s role and form the industry standard, ORAS maintainers proposed a new artifact manifest type to describe and query relationships between objects stored in a registry, without mutating the existing content.
+
+Initially, the reference types work was incubated under the CNCF [ORAS Artifact manifest](https://github.com/oras-project/artifacts-spec) project. It has been contributed to the OCI Image and Distribution v1.1 specifications in Sep 2022. Now it forms the industry standard and has been adopted by some cloud providers and open-source projects, such as [Azure Container Registry](https://azure.microsoft.com/en-us/products/container-registry) and [Zot registry](https://zotregistry.io/v1.4.3/).
+
+## Diverse evangelism and advocacy
+
+Open-source contributions are not limited to coding. The non-code contributions like blogging, writing documentation, and technical sharing are also important for the ORAS community. It’s so good to see more and more users and contributors from different organizations sharing their use cases and best practices with ORAS toolings via blog posts or conference presentations this year. You can learn more about their experience from their articles and videos below.
+
+### Blogs
+
+- [Notation signatures as ORAS and OCI artifacts](https://notaryproject.dev/blog/2022/oras-oci-artifacts-notation-signatures/) by [maxgio92](https://github.com/maxgio92) from Clastix
+- [Announcing Docker Hub OCI Artifacts Support](https://www.docker.com/blog/announcing-docker-hub-oci-artifacts-support/) by [MILOS GAJDOS](https://www.docker.com/author/milos-gajdos/) from Docker
+- [ORAS 0.14 and Future: Empower Container Secure Supply Chain](https://oras.land/blog/oras-0.14-and-future/) by Feynman Zhou from Microsoft
+- [ORAS 0.15: A Fully Functional OCI Registry Client](https://oras.land/blog/oras-0.15-a-fully-functional-registry-client/) by Feynman Zhou and Yi Zha from Microsoft
+- [Deploy OCI artifacts and Helm charts the GitOps way with Config Sync](https://cloud.google.com/blog/products/containers-kubernetes/gitops-with-oci-artifacts-and-config-sync) from Google Cloud blogs
+- [Storing ABAP build artifacts in OCI registry](https://blogs.sap.com/2022/07/26/storing-abap-build-artifacts-in-oci-registry/) by [Lars Hvam](https://people.sap.com/lars.hvam) from the SAP community blog
+
+### Presentations at conferences
```

Comment:
Might you please add the below entry to the "Presentations at conferences" section:

```
[Unleashing the Power of the Container Registry](https://www.youtube.com/watch?v=6fohG10y3z0&t=2002s) at DevConf.us by Andrew Block & Alex Flom
```
cc @sabre1041

Reply:
@afflom Good point. Added to the list above.

Reply:
Thanks so much! @FeynmanZhou